src/Aspire.Hosting.Elasticsearch/ElasticsearchContainerImageTags.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Elasticsearch;
+
+internal static class ElasticsearchContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "elasticsearch";
+    public const string Tag = "8.8.0";
```

Comment:
1. This version is a year old. There are newer ones.
2. We try to use `major.minor` versions if possible, so we automatically get patch updates without needing to change our code. Does Elasticsearch support these kinds of tags, so this doesn't get out of date so quickly?

Reply:
Seems so, I couldn't find any `major.minor` only tags. We'll just have to live with this. In this case though we'll want `8.8.2`